cmd/server.go

```diff
@@ -56,7 +58,13 @@ var cmdServer = &cobra.Command{
 			server.WithStore(repository),
 			server.WithStreamBroker(streamBroker),
 			server.WithConditionDefinitions(app.Config.ConditionDefinitions),
-			server.WithAuthMiddlewareConfig(app.Config.APIServerJWTAuth),
+		}
+
+		if viper.GetViper().GetBool("oidc.enabled") {
```

Comment:
Can we keep the configuration loading within the app package https://github.com/metal-toolbox/conditionorc/blob/ce8c4a67b0be44b411a40c959666e2ea8f58d395/internal/app/config.go#L118